Yurii_Telishevskyi/todo/src/app/app.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'todolist',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+  
```

Comment:
Extra line. Try to write code without extra lines/whitespaces (read the documentation about clean code)